ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -86,6 +86,10 @@
     private static final By DELETE_PATIENT_CANCEL  = By.cssSelector("#delete-patient-creation-dialog > div.dialog-content > button.cancel");
     private static final By EDIT_PATIENT_LINK = By.cssSelector("a[id='application.registrationapp.summary.editPatientLink']");
     private static final String REASON = "patient nolonger needed";
+    private static final By EDIT_VITALS_ICON = By.cssSelector("#encountersList i.editEncounter.edit-action.icon-pencil");
+    private static final By VIEW_VITALS_ICON  = By.cssSelector("#encountersList i.viewEncounter.view-action.icon-file-alt");
+    private static final By DELETE_VITALS_ICON = By.cssSelector("#encountersList i.deleteEncounterId.delete-action.icon-remove");
+    private static final By COMFIRM_DELETE_BUTTON  = By.cssSelector("#delete-encounter-dialog button.confirm.right");
```

Comment:
Are these elements on ClinicianFacingPatientDashboardPage or PatientVisitsDashboardPage?

```js
private static final By EDIT_VITALS_ICON = By.cssSelector("#encountersList i.editEncounter.edit-action.icon-pencil");
    private static final By VIEW_VITALS_ICON  = By.cssSelector("#encountersList i.viewEncounter.view-action.icon-file-alt");
    private static final By DELETE_VITALS_ICON = By.cssSelector("#encountersList i.deleteEncounterId.delete-action.icon-remove");
```

Reply:
> @Irenyak1 have mercy with the grand parent and take back the grand sons to their father, imagine all the grand sons of ClinicianFacingPatientDashboardPage are hosted at his little house, don't you think it will be overwhelming to him?

@kdaud  I definitely did not want to have these grand sons at their grand father's house but seems the father does not want to take the responsibility! :) . Man I tried to implement such workarounds before just to make sure the  PatientVisitDashboardPage serves its purspose but all was in vain. However, let me try again with your suggestion and see it will succeed.. Thanks.

Reply:
But Actually I am waiting for your feedback about the sync.